### ISSUE TITLE
Flameshot not open in Archlinux with kde

### ISSUE THREAD

**Issue by guilhermeaugusto9** [flameshot-org/flameshot#1594]
I have the icon on menu bar but when I click nothing happens,I tried to use the flameshot & and others codes I found on internet but nothing, folows my sistem data:

![arch](https://user-images.githubusercontent.com/23098357/116435070-5c3fbc00-a821-11eb-902e-e2414cfb945f.png)

**Comment by mmahmoudian** [comment]
Test this:
1. Open two terminals
2. In one terminal run `pkill flameshot ; flameshot`
3. In the other terminal run `flameshot gui`

Does it open? If not, what do you see in each terminal (send screenshot for example using the spectacle tool in KDE)

**Comment by guilhermeaugusto9** [comment]
thats what shows

![arch2](https://user-images.githubusercontent.com/23098357/116545047-4cc67e80-a8c6-11eb-99b4-2fd02dfcb180.png)

**Comment by guilhermeaugusto9** [comment]
I've tried `QT_QPA_PLATFORM=xcb flameshot  ` and similars ahah but nothing cold fix this problem.

**Comment by nullobsi** [comment]
the qt.qpa.wayland error is unrelated, and happens to a lot of qt applications on wayland that i know of

since you get no error, could you pastebin the output of `dbus-monitor --session sender=org.flameshot.Flameshot` to help figure out where flameshot may be waiting on? thanks :)

**Comment by mmahmoudian** [comment]
To clarify, what @nullobsi meant in simple words:
1. open 3 terminals
2. kill Flameshot if it is already open using `pkill flameshot`
3. in the first terminal run `dbus-monitor --session sender=org.flameshot.Flameshot`
4. in the second terminal run `flameshot`
5. in the third terminal run `flameshot gui`
6. copy the content of the first terminal in pastebin: https://bin.snopyta.org/
7. post the link here

**Comment by borgmanJeremy** [comment]
I just moved my main development box (arch kde) to wayland and can replicate this issue too.

**Comment by guilhermeaugusto9** [comment]
Here, and thank you so much for help I really appreciate

[](https://bin.snopyta.org/?3942422d230f4137#vqSmykVBWnjCWhG9yaCe8Jv4Ti2MJhdrbpEXFR2W7Fd)

**Comment by nullobsi** [comment]
now we know flameshot is using the kde portal to take a screenshot. could you try the same steps with:

`dbus-monitor --session sender=org.kde.KWin destination=org.kde.KWin`

instead? that should help pinpoint if KDE has an error/flameshot misses the reply

**Comment by guilhermeaugusto9** [comment]
Follows the code :
[https://bin.snopyta.org/?f2f148f7c65de030#4Tn7AqTs9zUtBAZJ2tFefuZDvy9mTMQemqotNv2XcYD4 (](https://bin.snopyta.org/?f2f148f7c65de030#4Tn7AqTs9zUtBAZJ2tFefuZDvy9mTMQemqotNv2XcYD4 )

**Comment by nullobsi** [comment]
well, there's our hit: it says the process is not authorized to take a screenshot

I think this is a duplicate of https://github.com/flameshot-org/flameshot/issues/1380 ? There is already a PR that should fix this

**Comment by borgmanJeremy** [comment]
Yeah this is a duplicate. Im fixing the merge issues with that PR on #1603.

**Comment by guilhermeaugusto9** [comment]
Sorry about my misunderstood but you will fix in another commit to the source code? I have to reinstall?

**Comment by borgmanJeremy** [comment]
Yeah it was fixed in that Pull Request. Since you are on Archlinux you can install flameshot-git from the AUR to run the latest commit on our main branch, or you need to wait for the 0.10.0 release.

**Comment by guilhermeaugusto9** [comment]
Yeah I reinstall and it is working, thank you so much for your help, I really appreciate you guys. Flameshot is essential for me and you  helped a lot !
